### PR TITLE
Make console-backend able to filter target node

### PR DESCRIPTION
**Pull request by nielsm5**

### CODE REVIEW

File: console/backend/src/main/java/org/frankframework/console/controllers/socket/FrankApiWebSocketBase.java [hunk 1]
```diff
@@ -67,23 +88,24 @@ protected final void propagateAuthenticationContext(String name) {
 	}
 
 	@Nullable
-	protected String compareAndUpdateResponse(RequestMessageBuilder builder) {
+	protected String compareAndUpdateResponse(RequestMessageBuilder builder, UUID target) {
 		final Message<?> response;
 		try {
-			response = gateway.sendSyncMessage(builder.build());
+			//TODO create a local cache and target specific members
```
Comment (Matthbo) [inline console/backend/src/main/java/org/frankframework/console/controllers/socket/FrankApiWebSocketBase.java:94]: Can't MessageCacheStore expanded to work for this?

File: management-gateway/src/main/java/org/frankframework/management/gateway/HazelcastInboundGateway.java [hunk 2]
```diff
@@ -52,9 +55,13 @@ public class HazelcastInboundGateway extends MessagingGatewaySupport {
 	private final String requestTopicName = HazelcastConfig.REQUEST_TOPIC_NAME;
 	private ITopic<Message<?>> requestTopic;
 
+	@Value("${instance.name:}")
```
Comment (Matthbo) [inline management-gateway/src/main/java/org/frankframework/management/gateway/HazelcastInboundGateway.java:58]: Does the `:` do something special?

**Review by evandongen — APPROVED** [review]

File: management-gateway/src/main/java/org/frankframework/management/gateway/HazelcastInboundGateway.java (hunk 2 above)
Comment (nielsm5) [reply management-gateway/src/main/java/org/frankframework/management/gateway/HazelcastInboundGateway.java:58]: Ensures the default value is empty and not null.

File: console/backend/src/main/java/org/frankframework/console/controllers/socket/FrankApiWebSocketBase.java (hunk 1 above)
Comment (nielsm5) [reply console/backend/src/main/java/org/frankframework/console/controllers/socket/FrankApiWebSocketBase.java:94]:
```suggestion
```

**Review by Matthbo — APPROVED** [review]

**Pull request merged**